mobify/progressive-web-scaffold
# Track is logged in in state

This PR adds isLoggedIn to the state and replaces hardcoded references to a user being logged in with the new isLoggedIn bool. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1111
 **Linked PRs**: #292

## Changes
- Added isLoggedIn to the app state
- Added selector for isLoggedIn
- Updated checkout-header to use isLoggedIn
- Updated checkout-shipping API calls to use isLoggedIn

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart
- navigate to the shipping page 
- everything should work as expected
- repeat as a logged in user
- everything should still work as expected (ex. shipping method should load, the header should not include a link to sign in)
